R/http.R

```diff
@@ -39,17 +42,26 @@ function(
     
     # generate request signature
     d_timestamp <- format(Sys.time(), "%Y%m%dT%H%M%SZ", tz = "UTC")
-    url <- paste0("https://comprehend.",region,".amazonaws.com")
+    url <- paste0("https://", service, ".",region,".amazonaws.com")
+    
+    if (service == "comprehend") {
```

Comment:
Same thing here, it would be more standard to use `match.arg` and a `switch`. You could also use `c("comprehend", "comprehendmedical")` as default value. Only the first value will be used by `match.arg` and it has the advantage of making valid `service` values very obvious without reading the manual.